lib/thrift/binary/framed/client.ex

```diff
@@ -155,24 +155,39 @@ defmodule Thrift.Binary.Framed.Client do
     {:connect, :reconnect, %{s | sock: nil}}
   end
 
-  @spec oneway(pid, data) :: :ok
+  @spec oneway(pid, String.t, data, options) :: :ok
   @doc """
   Execute a one way RPC. One way RPC calls do not generate a response,
   and as such, this implementation uses `GenServer.cast`.
   The data argument must be a properly formatted Thrift message.
   """
-  def oneway(conn, data) do
-    Connection.cast(conn, {:oneway, data})
+  def oneway(conn, rpc_name, serialized_args, _opts) do
+    seq_id = :erlang.unique_integer([:positive])
+    message = Binary.serialize(:message_begin, {:oneway, seq_id, rpc_name})
+    :ok = Connection.cast(conn, {:oneway, [message | serialized_args]})
+    :ok
   end
 
-  @spec request(pid, data, options) :: protocol_response
+  @spec call(pid, String.t, data, options) :: {:ok, data}
   @doc """
```

Comment:
Does this conflict with Connection.call? I don't see a reason for the name change; request described what it did. 
Also, the return value of the spec seems wrong. This can't fail?